File: 프로그래머스/unrated/181902. 문자 개수 세기/문자 개수 세기.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(string my_string) {
    vector<int> answer(52,0);
    for (int i = 0; i < my_string.size(); i++)
    {
        char cur = my_string[i];
        if ('a' <= cur && 'z' >= cur)
            answer[cur - 'a' + 26]++;
        else if ('A' <= cur && 'Z' >= cur)
            answer[cur - 'A']++;
    }
    return answer;
}

File: 프로그래머스/0/181902. 문자 개수 세기/문자 개수 세기.java
class Solution {

    public int[] solution(String my_string) {
        int[] arr = new int[52];
        for (int i = 0; i < my_string.length(); i++) {
            char charAt = my_string.charAt(i);
            if (Character.isUpperCase(charAt)) {
                arr[charAt - 65]++;
            } else { 
                arr[charAt - 71]++;
            }
        }
        return arr;
    }
}